src/main/webapp/i18n/en/programmingExercise.json

```diff
@@ -42,6 +42,7 @@
                 "deleteParticipationsSubmissionsAndResults": "Delete all student participations, submissions and results (on Artemis).",
                 "artefactsWarning": "Deleting student participations, submissions, and results without removing the associated repositories and build plans may lead to undeleted artifacts on {{ vcPlatform }} and {{ ciPlatform }}. Please carefully review if this is the desired action before proceeding.",
                 "recreateBuildPlans": "Recreate the exercise build plans (BASE and SOLUTION) again from the latest exercise template, overwriting any previous changes.",
+                "recreateCustomizedBuildPlans": "Recreate the customized build plans from the latest exercise template, overwriting any previous changes.",
```

Comment:
How can customized build plans be recreated? To which version of the customization? 

I guess we are resetting the build plans here to a default version? This translations seems to be misleading to me - what exactly happens here?

Reply:
I updated the translation to be clearer 👍